src/app/parcels/getExpandedParcelDetails.ts

```diff
@@ -190,9 +199,27 @@ export const formatBreakdownOfChildrenFromFamilyDetails = (
     return childDetails.join(", ");
 };
 
+export const processEventsDetails = (
+    events: Pick<Schema["events"], "event_data" | "event_name" | "timestamp">[]
+): EventTableRow[] => {
+    const eventTableRows: EventTableRow[] = [];
+
+    for (const event of events) {
+        eventTableRows.push({
+            eventInfo: `${event.event_name}` + (event.event_data ? ` (${event.event_data})` : ""),
+            timestamp: new Date(event.timestamp),
+        });
+    }
```

Comment:
```suggestion
    return events.map((event) => ({
        eventInfo: event.event_name + (event.event_data ? ` (${event.event_data})` : ""),
        timestamp: new Date(event.timestamp),
    }));
```

We can use `map` to do the exact same thing here :)

These are some of the most useful methods on JS arrays that might be interesting to look at:
- `.map`
- `.forEach`
- `.filter`
- `.reduce`

Reply:
Thanks!